geomesa-features/geomesa-feature-kryo/src/main/scala/org/locationtech/geomesa/features/kryo/impl/ActiveDeserialization.scala

```diff
@@ -6,38 +6,44 @@
  * http://www.opensource.org/licenses/apache2.0.php.
  ***********************************************************************/
 
-package org.locationtech.geomesa.features.kryo.impl
+package org.locationtech.geomesa.features.kryo
+package impl
```

Comment:
What cool thing does this do again?

Reply:
this brings the `kryo` package into scope, so you can use the package object and classes without importing them. if you're interested, more details here: https://www.artima.com/scalazine/articles/chained_package_clauses_in_scala.html